test/scripts/setupStargate.ts

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+
```

Comment:
Is tehre a way of executing `npm run prestart` from the ts-file? Then we could save this file!

Reply:
This is the file being replaced by your npm package? then in `package.json` we can run your cli command instead of calling this script

Reply:
I have an idea and probably know how to solve it.